utils/SnapPy/Snappy/MeteorologyCalculator.py

```diff
@@ -69,28 +69,40 @@ def __str__(self):
 class MeteorologyCalculator(abc.ABC):
     '''Base-class to pre-calculate/extract meteorology'''
     @staticmethod
-    def findGlobalData(res: GlobalMeteoResource, dtime: datetime):
-        '''Method to find the closest global dataset earlier than dtime.
+    def findAllGlobalData(res: GlobalMeteoResource): 
+        '''Static method to find the all global dataset
 
         Args:
-            res: resources from getGlobalMeteoResources
             dtime: datetime object with a start-time, which should be included in the dataset
+            count: number of file
 
         Returns:
-            A tuple with referencetime and filename
-
-        Raises:
-            MeteoDataNotAvailableException: no data for the dtime can be found
+            A list of tuples with [(forecast-time, file)]
```

Comment:
I consider types still to unstable in python 3.6, which we still have to use. Let's start with that when we use python 3.10, where it seems to be settled (PEP585), and then it's generic list,tuple instead of typing.List/typeing.Tuple.